source/guide_zola.rst

```diff
@@ -0,0 +1,216 @@
+.. highlight:: console
+
+.. author:: Kaisa-Marysia <https://www.kasiandras-dreams.de>
+
+.. tag:: rust
+.. tag:: blog
+.. tag:: cms
+
+####
+Zola
+####
+
+.. tag_list::
+
+Zola is an in rust written static site generator similar to Hugo and use the Tera template engine.
+
+----
+
+License
+=======
+
+Zola is released under the `MIT License <https://github.com/getzola/zola/blob/master/LICENSE>`_.
+
+Sources
+=======
+
+  * Documentation_
+  * Github_
+  * Zola_
+
+Prerequisites
+=============
+
+Zola is distributed as a single binary and requires ``GLIBC 2.28``, which isn't available in U7. We need to compile Zola from source with ``GLIBC 2.17``. Also, the default ``Cargo`` configuration will run into **Out Of Memory** during building the binary.
+
+Build from Source
+=================
+
+Create working directory
+------------------------
+
+First create a working directory in your home and navigate there.
+
+::
+
+  [isabell@stardust ~]$ mkdir ~/git
+  [isabell@stardust ~]$ cd ~/git
+
+Clone Repository
+----------------
+
+We need to clone the source code from GitHub, so we may build the binary file.
+
+::
+
+  [isabell@stardust git]$ git clone https://github.com/getzola/zola.git
+  [isabell@stardust git]$ cd zola
+
+Modify the configurations
+------------------------
+
+We need to change the `Link-time optimization (LTO) <https://nnethercote.github.io/perf-book/build-configuration.html#link-time-optimization>`_ from true to false, else the building process will fail because of a SIGKILL.
+Also, we need to set some cargo configs, which allows allocating more memory for the building process.
+
+::
+
+  [isabell@stardust zola]$ sed -i 's/lto = true/lto = false/g' Cargo.toml
+  [isabell@stardust zola]$ echo -e -n "[target.nightly-x86_64-unknown-linux-gnu]\nrustflags = [\"-C\", \"target-feature=+atomics,+bulk-memory,+mutable-globals\", \"-C\", \"link-arg=--max-memory=8589934592\"]" > ~/.cargo/config
```

Comment:
This does not work at this point for me, there is no ~/.cargo/config. Also please check the [Styleguide](https://github.com/Uberspace/lab/blob/main/STYLE.md) and describe what to edit where in the config files. Your approach might be convenient but we want our users to understand what's going on and this is not self explaining :)

Reply:
I modified the guide and removed the inplace changes with like in the style guide